client/app/components/composites/ListingCard/ListingCard.story.js

```diff
@@ -159,25 +160,38 @@ storiesOf('Search results')
         containerStyle
       ))
   ))
-  .add('ListingCard - image fail', () => (
-      r(storify(
-        r(ListingCard, Object.assign({},
-          {
-            id: 'lkjg84573874yjdf',
-            title: 'Picture load fails',
-            listingURL: 'http://marketplace.com/listing/342iu4',
-            imageURL: 'http://failingimage.com/image.png',
-            noImageText: 'No picture',
-            avatarURL: 'http://placehold.it/40x40',
-            profileURL: '#profile1',
-            price: 199,
-            priceUnit: '€',
-            distance: 9,
-            distanceUnit: 'km',
-            color: '#347F9D',
-            className: css.listing,
-          },
-        )),
-        containerStyle
-      ))
-  ));
+  .add('ListingCard - image fail', () => {
+    const story =
+      r(ListingCard, Object.assign({},
+        {
+          id: 'lkjg84573874yjdf',
+          title: 'Picture load fails',
+          listingURL: 'http://marketplace.com/listing/342iu4',
+          imageURL: 'http://failingimage.com/image.png',
+          noImageText: 'No picture',
+          avatarURL: 'http://placehold.it/40x40',
+          profileURL: '#profile1',
+          price: 199,
+          priceUnit: '€',
+          distance: 9,
+          distanceUnit: 'km',
+          color: '#347F9D',
+          className: css.listing,
+        },
+      ));
+
+    specs(() => describe('Failing image', function() {
+      const output = mount(story);
+      it('Should display "No picture"', function() {
+        expect(output.text()).to.include('No picture');
```

Comment:
This is async test and the version of jsdom Enzyme includes ("jsdom": "^6.1.0"), doesn't know how to deal with image loading. Even if we had a newer version, I'm not sure we want to set jsdom to fetch images (it most likely makes these test slow).